src/structured-data/schema.js

```diff
@@ -0,0 +1,64 @@
+export const schemaContext = { "@context": "http://schema.org" }
```

Comment:
can we name the functions to `get***` ? 

Ex: `getSchemaContext`, `getSchemaType` and so on.. Helps in a readability perspective